### PR TITLE
Removing testers from multiple groups

### PR DESCRIPTION
**Pull request by mpirri**

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a changelog summary in the Pull Request description @mpirri</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: pilot/lib/pilot/tester_manager.rb [hunk 1]
```diff
@@ -92,26 +93,30 @@ def list_testers(options)
 
     # private
```
Comment (snatchev) [inline pilot/lib/pilot/tester_manager.rb]: Looks methods below this can indeed be private. Is that something we want to enforce by uncommenting this line?

**Comment by snatchev** [comment]
Nice refactor! Looks like a few rubocop things came up, but otherwise LGTM.

File: pilot/lib/pilot/tester_manager.rb (hunk 1 above)
Comment (ohayon) [inline pilot/lib/pilot/tester_manager.rb]: I think we could? I wonder why it was commented out in the first place?

**Review by snatchev — APPROVED** [review]
LGTM

**Pull request merged**